region/src/main/java/org/motechproject/nms/region/service/impl/LocationServiceImpl.java

```diff
@@ -28,14 +30,21 @@
 @Service("locationService")
 public class LocationServiceImpl implements LocationService {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(LocationServiceImpl.class);
+
     private static final String INVALID = "<%s - %s : Invalid location>";
     private static final String STATE = "StateID";
```

Comment:
Since the other constants were changed to be _ID should the same be done for STATE and DISTRICT?